src/components/InviteToOpenSauced/InviteToOpenSaucedButton.ts

```diff
@@ -0,0 +1,20 @@
+import logoIcon from "../../assets/opensauced-icon.svg";
+import "../../index.css";
+import { createHtmlElement } from "../../utils/createHtmlElement";
+
+export const InviteToOpenSaucedButton = () => {
+  const inviteToOpenSaucedButton = createHtmlElement("a", {
+    className:
+      "inline-block mt-4 mb-1 text-white rounded-md p-2 no-underline text-md font-semibold text-center select-none w-full border border-solid cursor-pointer bg-[#21262d] hover:bg-[#ED5432]",
```

Comment:
If you think [#ED5432] and [#21262d] are colors we might use often, you can add them to tailwind config too. If they are common colors, we can just use one of tailwind's defaults.

Reply:
Alright. I'll do that.